membrane_curvature/lib/mods.py

```diff
@@ -2,19 +2,33 @@
 import numpy as np
 
 
+def get_positions(index):
```

Comment:
I don't think this function should be needed. See if you can write your code with numpy arrays and broadcasting/slicing.